tests/driver/nodejs/nodejs.rs

```diff
@@ -56,14 +56,15 @@ pub fn test(testcase: &test_driver_lib::TestCase) -> Result<(), Box<dyn Error>>
     let source = std::fs::read_to_string(&testcase.absolute_path)?;
     let include_paths = test_driver_lib::extract_include_paths(&source);
     for x in test_driver_lib::extract_test_functions(&source).filter(|x| x.language_id == "js") {
-        write!(main_js, "{{\n    {}\n}}\n", x.source.replace("\n", "\n    "))?;
+        write!(main_js, "{{\n    {}\n}}\n", x.source.replace('\n', "\n    "))?;
     }
 
     let output = std::process::Command::new("node")
         .arg(dir.path().join("main.js"))
         .current_dir(dir.path())
         .env("SLINT_NODE_NATIVE_LIB", std::env::var_os("SLINT_NODE_NATIVE_LIB").unwrap())
         .env("SLINT_INCLUDE_PATH", std::env::join_paths(include_paths).unwrap())
+        .env("SLINT_STYLE", std::env::var_os("SLINT_STYLE").unwrap_or_else(|| "fluent".into()))
```

Comment:
In the Rust driver (`build.rs`) we have this:

```rust
    //Make sure to use a consistent style
    println!("cargo:rustc-env=SLINT_STYLE=fluent");
```

Maybe it would be good if all these were in sync, i.e. default to fluent but allow override. Or do we want to allow an override?

Reply:
the interpreter driver has
```
    compiler.set_style(String::from("fluent")); // force to fluent style as Qt does not like multi-threaded test execution
```

But for C++ and node we currently don't enforce style.  In fact, i'm pertty sure we had test faillure because the Qt style was used. (which is good as it tests the Qt style)

Reply:
Our styles are rather different from each other. I made this default to fluent since that is what rust also uses (I missed the interpreter driver;-)... but I also want this to be configurable!

Our styles differ from each other, so we should have a way to test them all -- ideally we would do so in the CI.

Reply:
That's the point, the CI currently tests different style because c++ test and node tests used the native style while rust test uses the fluent style

Reply:
Yes, but ideally we would have coverage for all styles in all languages.

Reply:
I mean yeah, ideally we should test an exponential amount of things. But we're limited by resources. And the problem in the styles are likely to be problem in each language anyway.